src/components/ArticlePage/index.js

```diff
@@ -0,0 +1,117 @@
+
+class ArticlePage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      article: {
+        id: 1,
+        title: 'Ronaldo crowned world best',
+        category: 'Sport',
+        imageUrl: 'https://images.unsplash.com/photo-1527871252447-4ce32da643c6?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2689&q=80',
+        author: {
+          fullName: 'Habeeb Olawale',
+          avatarUrl: 'https://res.cloudinary.com/vivavalinor/image/upload/v1541446448/nql15siqfajqpsrudk7b.jpg'
+        },
+        comments: [{
+          id: 1,
+          date: '12 November, 2018',
+          body: comment,
+          author: {
+            fullName: 'Chris Akanmu',
+            avatarUrl: 'https://res.cloudinary.com/vivavalinor/image/upload/v1541446448/nql15siqfajqpsrudk7b.jpg'
+          }
+        }, {
+          id: 2,
+          date: '13 November, 2018',
+          body: comment,
+          author: {
+            fullName: 'Abiodun Abudu',
+            avatarUrl: 'https://res.cloudinary.com/vivavalinor/image/upload/v1541446448/nql15siqfajqpsrudk7b.jpg'
+          }
+        }]
+      }
+    };
+  }
+
+  render() {
+    const { article } = this.state;
+    const { author } = article;
+    return (
+      <div className="site-content">
+        <p>Header here</p>
+        <div className="article-image-container">
+          <i id="back-btn" className="fa fa-arrow-circle-o-left fa-3x" />
+          <i id="bookmark-btn" className="fa fa-bookmark fa-2x" />
+          <img src={article.imageUrl} alt="article" />
+        </div>
+        <div className="article-content">
+          <span id="article-category">{article.category}</span>
+          <span id="article-title">{article.title}</span>
+          <div className="article-main">
+            <div className="author-container">
+              <img src={author.avatarUrl} alt="author" />
+              <div>
+                <span id="author-name">{author.fullName}</span>
+                <div>
+                  <span id="author-date">11, November 2018</span>
+                  •
+                  <span id="read-time">2 min read</span>
+                </div>
+              </div>
+            </div>
+            <div className="article-body">
+              <p>
+                {body}
+                <br />
+                <br />
+                {body}
+                <br />
+                <br />
+                {body}
+                <br />
+                <br />
+                {body}
+              </p>
+            </div>
+            <div className="article-options">
+              <div className="article-share-options">
+                <i className="fa fa-bookmark-o fa-2x" />
+                <i id="goolge-icon" />
+                <i id="facebook-icon" />
+                <i id="twitter-icon" />
+              </div>
+              <div className="article-like-options">
+                <i className="fa fa-thumbs-o-up fa-2x" />
```

Comment:
I think the javascript to have this changing color/class onClick should be included.

Reply:
I've added logic state logic to manage the icon class. I think the onClick for this should fire off a reducer that will change the state and then the icon's class, so I can't write that yet as it's beyond the scope of this task.

Reply:
Update: I have now added all the click handlers for the buttons. They all now work.